src/common/components/repository-row/index.js

```diff
@@ -14,7 +14,13 @@ const RepositoryRow = (props) => {
     <Row>
       <Data col="30"><Link to={ `/${fullName}/builds` }>{ fullName }</Link></Data>
       <Data col="20"> </Data>
```

Comment:
This is the column that checks snapcraft.yaml status.
If `snapName` is defined, we know that snapcraft.yaml is valid, so we can show it here.